client/verta/verta/deployment.py

```diff
@@ -14,6 +14,21 @@
 from ._internal_utils import _utils
 
 
+# TODO: move these to verta/deployment/strategies if restructuring
```

Comment:
Why don't we do this right now? Seems like a small effort now vs larger effort later, no?

Reply:
Sure! I can spend some time doing this.